src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -126,28 +126,69 @@ cdef class SSLSessionCacheLRU:
     grpc_shutdown()
 
 
+cdef int verify_peer_callback_wrapper(const char* servername, const char* cert, void* userdata) with gil:
```

Comment:
Why `int` for this return type?

Reply:
This is matching the required signature of the new callback function. There's some details in the header [here](https://github.com/grpc/grpc/blob/master/include/grpc/grpc_security.h#L169).